contrib/agent/src/main/java/io/opencensus/contrib/agent/instrumentation/ContextStrategyImpl.java

```diff
@@ -67,7 +68,7 @@ public void attachContextForThread(Thread thread) {
       Context context = savedContexts.getIfPresent(thread);
       if (context != null) {
         savedContexts.invalidate(thread);
-        context.attach();
+        Preconditions.checkNotNull(context.attach(), "context"); // Work around findbugs warning
```

Comment:
I think we should not do this just for findbugs (this is very annoying and code looks bad). Please rewrite this as if context == null return.

Reply:
Got it, done.